.github/workflows/ci.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           php-version: '7.2'
           coverage: none
           extensions: mbstring, intl
+          ini-values: error_reporting=E_ALL
```

Comment:
can we do this in bootstrap instead, so direct running locally has this, too?

Reply:
eg https://github.com/laravel/framework/blob/v5.5.50/tests/bootstrap.php

Reply:
Good idea. It appears to work.